src/main/java/com/lgtm/easymoney/controllers/RequestController.java

```diff
@@ -25,11 +22,35 @@ public RequestController(UserService userService, RequestService requestService)
         this.requestService = requestService;
     }
 
+
     @PostMapping("/create")
-    // make a new request
     public ResponseEntity<RequestRsp> createARequest(@Valid @RequestBody RequestReq req) {
         return requestService.createARequest(req);
     }
-    // TODO handle other request routes
+
+    @GetMapping("/{uid}")
+    public ResponseEntity<RequestRsp> getRequests(@PathVariable(value="uid") Long uid) {
+        // get all requests that are sent/received by user
+        // param: uid (from/to are the same for now) we may add filtering features later
+        // return: list of requests
+        return requestService.getRequestsByUser(userService.getUserByID(uid));
+    }
+
+
+    @GetMapping("/accept")
+        public ResponseEntity<RequestRsp> acceptRequest(@RequestParam(name="rid")Long requestID,
```

Comment:
I think this should be PUT? might not be a good practice to have ids in the path. `@RequestBody Map<String, Long> json` is better